tests/test_workers/test_cleanup_job.py

```diff
@@ -69,6 +69,36 @@
     },
 }
 
+mock_failed = {
+    "items": [
+        {
+            "dependencies": 309,
+            "environment_variables": {},
+            "flags": [],
+            "id": 52,
+            "pkg_managers": ["gomod"],
+            "ref": "a7ac8d4c0b7fe90d51fb911511cbf6939655c877",
+            "repo": "https://github.com/kubernetes/kubernetes.git",
+            "state": "failed",
+            "state_reason": "The request failed",
+            "updated": "2019-09-05T18:24:50.857861",
+            "user": "mprahl@redhat.com",
```

Comment:
Exposing my work email to the public? 😛

Reply:
I didn't like it either but you have it at couple of more places in this file itself. so I just copied the mock object.
Do you want me to change them as well in a separate commit?

Reply:
Running `git blame --ignore-rev e746e8b3 --ignore-rev 0ea1a626 tests/test_workers/test_cleanup_job.py` shows the real culprit behind this.

I won't name names.... well maybe I will... @yashvardhannanavati :smile:

Reply:
@nirzari I'll make a PR to remove it on the existing code. Could you please replace `mprahl@redhat.com` with your favorite celebrity's fake email address (e.g. `tbrady@domain.local`)?

Reply:
https://github.com/release-engineering/cachito/pull/220

Reply:
@mprahl haha, you were the one who probably reviewed my PR's. But eh, it was intentional.

Reply:
@yashvardhannanavati you slipped one past the goalie! I'll be even stricter on the next IIB code review :wink:

Reply:
haha, let's see if you catch something like this in IIB now.